### DISCUSSION TITLE
Colorblind status icons

### DISCUSSION THREAD

**Discussion by GTcreyon** [stoatchat/discussions#379] [Feature Suggestions]
Something similar to Discord's moon, circle, and dash icons would be nice for colorblind people, as well as those who use reduced saturation.

**Comment by pinkjimrussiagirl** [comment]
> Something similar to Discord's moon, circle, and dash icons would be nice for colorblind people, as well as those who use reduced saturation.

is it icons like these
![IMG_6821](https://github.com/revoltchat/revolt/assets/111091825/8c588d4b-ab8a-4211-9e96-3d11ff38551c)

if it is i’m also trying to add these icons to my custom css theme

**Comment by DeclanChidlow** [comment]
This is something we are actively pursuing in the new web client.

Give it a look.
https://revolt.chat/posts/refreshing-revolt-web

**Comment by insertish** [comment]
This is slated to be released in the new client.
